app/lp/utils.ts

```diff
@@ -69,6 +72,33 @@ export default function usePool() {
     });
   }
 
+  /** AMBIENT REWARDS TIMER */
+  const [rewardTime, setRewardTime] = useState(0n);
+  const getRewardsTime = async (): Promise<bigint> => {
+    const blockNumber = await fetchBlockNumber({
```

Comment:
always wrap an **await** in tryCatch block as there's possibility for it to return error.